Examples/PublisherExampleSwiftUI/PublisherExampleSwiftUI/ViewModel/AddTrackableViewModel.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import SwiftUI
+import AblyAssetTrackingPublisher
+import AblyAssetTrackingCore
+
+class AddTrackableViewModel: ObservableObject {
+    @Published var trackableId = ""
+    @Published var resolutionMinimumDisplacement: String  = "\(SettingsModel.shared.defaultResolution.minimumDisplacement)"
+    @Published var resolutionDesiredInterval: String  = "\(SettingsModel.shared.defaultResolution.desiredInterval)"
+    @Published var setResolutionConstraints = false
+    
+    var resolutionAccuracy: String = SettingsModel.shared.defaultResolution.accuracy.rawValue
+    var accuracies: [String] {
+        [Accuracy.low,
+         Accuracy.high,
+         Accuracy.balanced,
+         Accuracy.maximum,
+         Accuracy.minimum].sorted().map(\.rawValue)
+    }
+    
+    var isValid: Bool {
+        !(trackableId.isEmpty || (setResolutionConstraints && customResolution == nil))
+    }
+    
+    private var customResolution: Resolution? {
+        guard let accuracy = Accuracy(rawValue: resolutionAccuracy),
+              let minimumDisplacement = Double(resolutionMinimumDisplacement),
+              let desiredInterval = Double(resolutionDesiredInterval) else {
+            return nil
+        }
+        
+        return .init(accuracy: accuracy, desiredInterval: desiredInterval, minimumDisplacement: minimumDisplacement)
+    }
+    
+    func createTrackable() -> Trackable {
```

Comment:
I don't think this method ever gets called. In `AddTrackableView`, when tapping on the `Add Trackable` button, you're creating a trackable instance with just an id, without using the selected resolution constraints:
```
let trackable = Trackable(id: viewModel.trackableId)
```

Reply:
Oops, good catch! Fixed it.